src/main/org/apache/tools/ant/types/XMLCatalog.java

```diff
@@ -197,10 +197,7 @@ public Path createClasspath() {
     }
 
     /**
-     * Allows simple classpath string.  Not allowed if this catalog is
-     * itself a reference to another catalog -- that is, a catalog
-     * cannot both refer to another <em>and</em> contain elements or
-     * other attributes.
+     * Append the contents of the other Path instance to this.
```

Comment:
This replaces a somewhat lengthy detailed description of what happens by a less clear one-liner. Why is this better?